docs/content/doc/setup/full-docker-example.md

```diff
@@ -70,6 +70,34 @@ You'll also need to change the `VIKUNJA_DATABASE_TYPE` to `postgres` on the api
 During this time, the api container will fail to start at all. It will automatically restart every few seconds.
 </div>
 
+## Sqlite
+
+Vikunja supports postgres, mysql and sqlite as a database backend. The examples on this page use mysql with a mariadb container.
+To use sqlite as a database backend, change the `api` section of the examples to this:
```

Comment:
I think we should highlight the two actual changes here, being the `VIKUNJA_DATABASE_PATH` and the `/db` volume. Otherwise, people will use the config below verbatim and complain about other things not working.

Reply:
Done below 🙏